src/ModelObjects/component.rs

```diff
@@ -177,6 +178,43 @@ impl Component {
         };
     }
 
+    pub fn get_all_edges_from(&self, location: &Location) -> Vec<&Edge> {
```

Comment:
This method isn't used currently but i think that's fine. We'll eventually need it if we do local maximal constant extrapolation so we can just keep it here.